src/loops/nested_loops/hard/q8/PrimeOrdial.java

```diff
@@ -0,0 +1,37 @@
+package loops.nested_loops.hard.q8;
+
+/*
+Write a program called PrimeOrdial that will keep asking users for a number until they
+enter a prime number, after which it will break from the loop.
```

Comment:
Typically, prime numbers are positive, so I think we should exclude negatives from this question and instead have an input check if the number is less than 0.

Reply:
There's no need to. Negative numbers cannot by definition be prime, and the program recognizes that. Negative numbers aren't prime and if you feed the program any negative number it will tell you it isn't prime. That seems fine to me.

Reply:
Ah ok. I was not certain if the code would recognize that a negative is not prime, but if it works then that is fine.